src/supertux/shrinkfade.hpp

```diff
@@ -24,7 +24,8 @@
 class ShrinkFade final : public ScreenFade
 {
 public:
-  ShrinkFade(const Vector& point, float fade_time);
+  enum Direction { FADEOUT, FADEIN };
+  ShrinkFade(const Vector& point, float fade_time, Direction = FADEOUT);
```

Comment:
Separating enumerators from constructor:

```suggestion
  enum Direction { FADEOUT, FADEIN };

public:
  ShrinkFade(const Vector& point, float fade_time, Direction = FADEOUT);
```